### DISCUSSION TITLE
Getting started with DeepSqueak

### DISCUSSION THREAD

**Discussion by jomi-doe** [DrCoffey/DeepSqueak#182] [Q&A]
This is the first time I open Deepsqueak (with no prior knowledge about GitHub, Matlab, Neuronal Networks etc...) and I followed all  steps you provided in the manual, but I get stuck in the section "USV Detection"

it states: "Before vocalizations can be detected, an [audio folder], neural network folder, and [output folder] must be selected."
I selected the audio folder which contains the .wav files i wanna analyze, i selected an output folder, but for the neuronal network folder I'm already kinda confused... 

- is this folder provided by DeepSqueak (since it contains a folder "\DeepSqueak-master\DeepSqueak-master\Networks") ? 
'Cause if i choose this folder and one of the neuronal networks (Long Rat Detector YOLO R1 or Mouse Detector Yolo R2 or Rat Detectos Yolo R1,) and one my audio files for processing and clicking "Detect calls" I get the error message saying:

"Error in Network Skipping Audio Chunk"

May you tell me what's going on? 
Or am i missing the point completely and I'm supposed to create and train my own neuronal network with my own files first? If yes, how do i do so? 

I'm really looking forward to analyze USV of our rats and hope you can help me on that
Thanks in advance!

**Comment by SarahBacc** [comment]
Hi @jomi-doe! Did you figure out what to do with the program?
I have also just started using it with no previous knowledge. From what I understood, we can just choose one of the networks that DeepSqueak provides. However, I am also getting the same error message "Error in network, skiping audio chunk" with all the audios I've tried analyzing.

**Comment by DrCoffey** [comment]
Hey @jomi-doe and @SarahBacc, that is a generic error if something fails during detection. It usually means there is some issue with either the audio file itself, or there is an issue with not installing all the required toolboxes. Can you detect the included example audio files?

**Comment by AsafKeller** [comment]
I have the same issue, and the error message appears also with the included example audio files. Running MatlabR2024a and DeepSqueak 3.1.0

**Comment by gushasha00-commits** [comment]
I have the same issue
